lighthouse-cli/test/smokehouse/pwa-expectations.js

```diff
@@ -78,6 +78,46 @@ module.exports = [
         score: false
       },
       'webapp-install-banner': {
+        score: false
+      },
+      'splash-screen': {
+        score: true
+      },
+      'themed-omnibox': {
+        score: true
+      },
+      // 'cache-start-url': {
+      //   score: true
+      // }
+    }
+  },
+
+  {
+    initialUrl: 'https://www.chromestatus.com/features',
```

Comment:
I did it twice, the only difference is `webapp-install-banner` XD
Or maybe a a smoketest is enough?